freqtrade/exchange/exchange.py

```diff
@@ -332,7 +332,8 @@ def validate_pairs(self, pairs: List[str]) -> None:
                 logger.warning(f"Pair {pair} is restricted for some users on this exchange."
                                f"Please check if you are impacted by this restriction "
                                f"on the exchange and eventually remove {pair} from your whitelist.")
-            if not self.get_pair_quote_currency(pair) == self._config['stake_currency']:
+            if (self._config['stake_currency'] and
+                    not self.get_pair_quote_currency(pair) == self._config['stake_currency']):
```

Comment:
why not `self.get_pair_quote_currency(pair) != self._config['stake_currency']`?

Reply:
well THAT is a different topic :D 
great catch :+1: